lib/fse.h

```diff
@@ -45,6 +45,32 @@ extern "C" {
 #include <stddef.h>    /* size_t, ptrdiff_t */
 
 
+/*-*****************************************
+*  FSE_PUBLIC_API : control library symbols visibility
+******************************************/
+#if defined(__GNUC__) && (__GNUC__ >= 4)
+#  define FSE_PUBLIC_API __attribute__ ((visibility ("default")))
```

Comment:
Do you mean that the exact same code will be sync'ed between this repository and zstd?

If yes, I changed the config option to be `DLL_EXPORT`, instead of `ZSTD_DLL_EXPORT`, so has to not conflict with libzstd if this was the case.

Maybe for better clarity, I should change it to `FSE_DLL_EXPORT` ?